Fix documentation of arguments for torch.nn.functional.Linear

mikaylagawarecki opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #66884

Addressing docs fix mentioned in issue #64978 on Github

Differential Revision: [D31767303](https://our.internmc.facebook.com/intern/diff/D31767303/
<img width="745" alt="Screen Shot 2021-10-19 at 2 29 59 PM" src="https://user-images.githubusercontent.com/35276741/137971953-759dab17-df24-4fbb-8d2a-c3a37d763b57.png">
)

Commit:
Fix documentation of arguments for torch.nn.functional.Linear

Addressing docs fix mentioned in issue 64978 on Github

Differential Revision: [D31767303](https://our.internmc.facebook.com/intern/diff/D31767303/)

[ghstack-poisoned]

Commit:
Update on "Fix documentation of arguments for torch.nn.functional.Linear"


Addressing docs fix mentioned in issue 64978 on Github

Differential Revision: [D31767303](https://our.internmc.facebook.com/intern/diff/D31767303/)

[ghstack-poisoned]

jbschlosser reviewed (commented):
Thanks for the update! Changes look great - just a couple tiny tweaks needed and we're good to go :)

Commit:
Update on "Fix documentation of arguments for torch.nn.functional.Linear"


Addressing docs fix mentioned in issue #64978 on Github

Differential Revision: [D31767303](https://our.internmc.facebook.com/intern/diff/D31767303/
<img width="745" alt="Screen Shot 2021-10-19 at 2 29 59 PM" src="https://user-images.githubusercontent.com/35276741/137971953-759dab17-df24-4fbb-8d2a-c3a37d763b57.png">
)

[ghstack-poisoned]

Commit:
Update on "Fix documentation of arguments for torch.nn.functional.Linear"


Addressing docs fix mentioned in issue #64978 on Github

Differential Revision: [D31767303](https://our.internmc.facebook.com/intern/diff/D31767303/
<img width="745" alt="Screen Shot 2021-10-19 at 2 29 59 PM" src="https://user-images.githubusercontent.com/35276741/137971953-759dab17-df24-4fbb-8d2a-c3a37d763b57.png">
)

[ghstack-poisoned]

jbschlosser reviewed (approved):
LGTM! :)
